sunpy/instr/aia/response.py

```diff
@@ -28,7 +29,7 @@
 
 import numpy as np
 import pandas as pd
-import astropy.units as u
+from astropy import units as u
```

Comment:
We try and use the `import astropy.units as u` form everywhere just for consistency.